bricks/r13n/hooks/post_gen.dart

```diff
@@ -6,10 +6,11 @@ import 'package:meta/meta.dart';
 
 Future<void> run(HookContext context) async {
   final runProcess = ProcessOverrides.current?.runProcess ?? Process.run;
-  final progress = context.logger.progress('Running "flutter format ."');
+  final arbDir = context.vars['arbDir'] as String;
+  final progress = context.logger.progress('Running "flutter format $arbDir"');
   final result = await runProcess(
     'flutter',
```

Comment:
No dart format?

Reply:
haha was trying to keep it simple but you're right...might as well 👍

Reply:
@wolfenrain just updated to use `dart format` 👍